src/tests/dictionaryConcepts/components/AnswersRow.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import AnswersRow from '../../../components/dictionaryConcepts/components/AnswersRow';
+import answers from '../../__mocks__/answers';
+import { existingConcept } from '../../__mocks__/concepts';
+
+const props = {
+  pathName: {
+    language: 'en',
+  },
+  answer: answers,
+  removeAnswer: jest.fn(),
+  removeDataFromRow: jest.fn(),
+  addDataFromAnswer: jest.fn(),
+  existingConcept: [existingConcept],
+  newRow: {
+    uuid: 4566,
+  },
+};
+
+describe('Test suite for AnswersRows ', () => {
+  it('should render AnswersRows  Component', () => {
+    const wrapper = shallow(<AnswersRow {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should handle remove', () => {
+    const wrapper = mount(<AnswersRow {...props} />);
+    expect(wrapper).toMatchSnapshot();
+    wrapper.find('.concept-form-table-link.answer').simulate('click');
+    wrapper.find('#remove-answer').simulate('click');
+  });
+
+  it('should handle change', () => {
+    const wrapper = mount(<AnswersRow {...props} />);
+    expect(wrapper).toMatchSnapshot();
+    const event = {
+      target: {
+        answer: '/users/nesh/sources/test/concepts/2140db25-4d4a-4804-90cc-6fd8328dfe9e/',
+      },
+    };
+    wrapper.find('.form-control.answer').simulate('change', event);
+  });
+
+  it('should handle componentDidUpdate', () => {
```

Comment:
I think you should refactor all your tests here to assert some functionality that simulates user interactions with the system.

Reply:
Yes, I agree. Will work on this!